tutorials/roofit/rf317_global_observables.C

```diff
@@ -0,0 +1,169 @@
+/// \file
+/// \ingroup tutorial_roofit
+/// \notebook -js
+/// This tutorial explains the concept of global observables in RooFit, and
+/// showcases how their values can be stored either in the model or in the
+/// dataset.
+///
+/// \macro_code
+///
+/// \date January 2022
+/// \author Jonas Rembser
+
+#include <RooArgSet.h>
+#include <RooConstVar.h>
+#include <RooDataSet.h>
+#include <RooFitResult.h>
+#include <RooGaussian.h>
+#include <RooProdPdf.h>
+#include <RooRealVar.h>
+
+// Helper function to call RooAbsPdf::fitTo.
+template <class... Args_t>
+std::unique_ptr<RooFitResult> doFit(RooAbsPdf &model, RooAbsData &data, Args_t &&...args)
+{
+   using namespace RooFit;
+
+   // we clone the model to not change the parameters by fitting
+   std::unique_ptr<RooAbsPdf> modelClone{static_cast<RooAbsPdf *>(model.cloneTree())};
+
+   return std::unique_ptr<RooFitResult>{modelClone->fitTo(data, PrintLevel(-1), Save(), std::forward<Args_t>(args)...)};
+}
+
+void rf317_global_observables()
+{
+   using namespace RooFit;
+
+   // Introduction
+   // ------------
+   //
+   // With RooFit, you usually optimize some model parameters `p` to minimize
+   // the negative log-likelihood `l` given the per-event or per-bin
+   // observations `x`:
+   //
+   // l( x | p )
```

Comment:
I find this notation confusing. I was going to suggest changing `+` to `*`, because I thought that `l == likelihood`. Maybe you use `NLL`?

Reply:
Ok, now I used `L` for likelihood all the way, use `*` and don't talk about negative logarithms.